ui/client/components/ProcessAttachments.js

```diff
@@ -40,20 +39,22 @@ export class ProcessAttachments_ extends React.Component {
         <ul className="process-attachments-list">
           {_.map(this.props.attachments, (attachment, idx) => {
             return (
-              <div key={idx}>
+              <div key={idx} className={"attachment-section"}>
                 <div className="download-attachment">
                   <div
                       className="espButton download-button"
                       dangerouslySetInnerHTML={{__html: InlinedSvgs.buttonDownload}}
                       onClick={this.downloadAttachment.bind(this, attachment.processId, attachment.processVersionId, attachment.id)}
                   />
                 </div>
-                <div className="header">
-                  <span className="label label-info">{attachment.user}</span>
-                  <span title={DateUtils.formatAbsolutely(attachment.createDate)} className="date">{DateUtils.formatRelatively(attachment.createDate)}</span>
-                  <p>{ProcessUtils.processDisplayName(attachment.processId, attachment.processVersionId)}</p>
+                <div className={"attachment-details"}>
+                  <div className="header">
+                  <span title={DateUtils.formatAbsolutely(attachment.createDate)} className="date">
```

Comment:
What do you think about new component to presenting Date?

Reply:
Done